Formula/tepl.rb

```diff
@@ -2,9 +2,19 @@ class Tepl < Formula
   desc "GNOME Text Editor Product Line"
   homepage "https://gitlab.gnome.org/swilmet/tepl"
   url "https://gitlab.gnome.org/swilmet/tepl.git",
-      tag:      "6.5.1",
-      revision: "606de78acb7a2b3268d59eb8e975bcac0ecf1691"
+      tag:      "6.4.0",
+      revision: "d1e9cc2e4a0738f68d9002f13724b7239f075844"
   license "LGPL-2.1-or-later"
+  revision 2
+  version_scheme 1
```

Comment:
Note that there's disagreement among maintainers about the use of `version_scheme` for reverting mistaken upgrades. CC @Homebrew/core

Reply:
What's the alternative?

Reply:
Bumping the revision far enough for it to be seen as an update I guess, or just waiting for an update

Reply:
How does bumping the revision take priority over the version field?

Reply:
Well, if we had a 6.4.0_2 before and you make it a 6.4.0_3 now, it'll be an update where possible. It's not a real fix though.

Reply:
Yes, we should not be using `version_scheme` for this. We should just revert and, if we need to, `revision`.